<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventoryActionDialog</class>
 <widget class="QDialog" name="InventoryActionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Book Title</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="titleLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>49</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Author</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="authorLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Publisher</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="publisherLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Location</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="locationLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stockLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="uploadButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Upload...</string>
   </property>
  </widget>
  <widget class="QLabel" name="coverImageLabel">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>50</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cover Image( NULL )</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>310</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="priceLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>220</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Price( in naira )</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>titleLineEdit</tabstop>
  <tabstop>authorLineEdit</tabstop>
  <tabstop>publisherLineEdit</tabstop>
  <tabstop>locationLineEdit</tabstop>
  <tabstop>priceLineEdit</tabstop>
  <tabstop>stockLineEdit</tabstop>
  <tabstop>uploadButton</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
